packages/common/src/viz/barChart.ts

```diff
@@ -0,0 +1,86 @@
+import { friendlyName } from '../types/field';
+import { type ResultsTransformerBase } from './base';
+
+export type BarChartStyling = {
```

Comment:
At least as this is set up, it's not really styling at the moment. It seems like this could all be part of the general BarChartConfig instead of splitting this out, no? Unless there is a plan to make this more separate kinds of properties.